components/NewsfeedCard/NewsfeedCardBody.tsx

```diff
@@ -0,0 +1,176 @@
+  const isMobile = useMediaQuery("(max-width: 550px)")
+  return (
+    <CardBootstrap.Body className={`p-0`}>
+      <div
+        className={`align-items-center bg-info d-flex mx-2 mt-1 mb-2 p-2 rounded`}
+      >
+        {isMobile ? (
+          <>
+            <Col className={`d-flex align-items-center flex-column m-2`}>
+              <img
+                src="\images\clock.svg"
+                height="105.6"
+                width="105.6"
+                alt={imgAltTxt}
+                className={`m-3`}
+              />
+              <CardBootstrap.Text className={`mb-0 mt-2`}>
+                <strong>{text}</strong>
+              </CardBootstrap.Text>
+              <>
+                {"Action taken on "}
+                {timestamp}
+              </>
+            </Col>
+          </>
+        ) : (
+          <>
+            <img
+              src="\images\clock.svg"
+              height="105.6"
+              width="105.6"
+              alt={imgAltTxt}
+              className={`m-3`}
+            />
+            <Col className={`m-2`}>
+              <CardBootstrap.Text className={`mb-0`}>
+                <strong>{text}</strong>
+              </CardBootstrap.Text>
+              <>
+                {"Action taken on "}
+                {timestamp}
+              </>
+            </Col>
+          </>
+        )}
+      </div>
+    </CardBootstrap.Body>
+  )
+}
+
+export const NewsfeedTestimonyCardBody = (props: NewsfeedCardBodyProps) => {
+  const { imgSrc, imgAltTxt, text, timestamp } = props
+  const isMobile = useMediaQuery("(max-width: 550px)")
+
+  function capitalizeFirstLetter(x: String | undefined) {
+    return String(x).charAt(0).toUpperCase() + String(x).slice(1)
+  }
+
+  let endorsementColor = ""
+
+  switch (imgAltTxt) {
+    case "endorse":
+      endorsementColor = `text-success`
+      break
+    case "oppose":
+      endorsementColor = `text-warning`
+      break
+    default:
+      endorsementColor = `text-secondary`
+      break
+  }
+
+  return (
+    <div>
+      <CardBootstrap.Body className={`p-2`}>
+        <CardBootstrap.Text className={`text-secondary mb-2`}>
+          {/* uncomment when bill text is made available to for results: type testimony */}
+          {/* <strong>{testimonyText}</strong> */}
+        </CardBootstrap.Text>
+        <Row className={`bg-body mx-0 mt-1 rounded`}>
+          {isMobile ? (
+            <Col className={`d-flex flex-column m-2 w-100`} xs="2">
+              <div className={`align-self-center my-3`}>
+                {imgSrc && (
+                  <img
+                    src={imgSrc}
+                    height="108"
+                    width="106.5"
+                    alt={imgAltTxt}
+                  ></img>
+                )}
+                <div className={`${endorsementColor} fs-5`}>
+                  {capitalizeFirstLetter(props.imgAltTxt)}
+                </div>
+              </div>
+              <strong>
+                {`"`}
```

Comment:
nit: Is there a more concise way to do this (maybe {`"${text}"`})?

Reply:
done